tams_ur5_setup_bringup/config/apriltags2/tags.yaml

```diff
@@ -0,0 +1,49 @@
+# # Definitions of tags to detect
```

Comment:
This is a config file, not a tutorial.
Please drop all the explanations. They belong in the examples of the `apriltags2_ros` package, but not here.